content/02.introduction.md

```diff
@@ -75,15 +75,15 @@ The current pandemic of COVID-19, caused by the severe acute respiratory syndrom
 The possibility of asymptomatic cases is also being investigated.
 In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes).
 Other modes of transmission, such as through touching surfaces or objects and then touching mucous membranes, can also be investigated.
-While droplet-based and contact were initially considered to be the primary modes of SARS-CoV-2 transmission [@doi:10.1038/s41368-020-0075-9], as additional information has emerged, aerosol transmission has become a growing concern [@10.1126/science.abc6197].
+While droplet-based and contact were initially considered to be the primary modes of SARS-CoV-2 transmission [@doi:10.1038/s41368-020-0075-9], as additional information has emerged, aerosol transmission has become a growing concern [@doi:10.1126/science.abc6197].
 Other aspects of transmission to investigate are the relationship between infectiousness and virus shedding with disease period or symptoms and also the proportions of cases that are attributable to various types of transmission events, such as transmission between relatives, nosocomial transmissions, and other possible types of interactions.
 Some information about these characteristics of transmission are available for the highly pathogenic coronaviruses SARS-CoV and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
 For SARS-CoV-2 it is still being investigated whether, in addition to being spread by people who show symptoms,  the virus can be transmitted by people who do not show symptoms [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html], such as during the pre-symptomatic stage of infection or in people with asymptomatic infections (a presentation which sometimes occurs in other infectious diseases but is still being investigated for SARS-CoV-2).
 
 #### Cell Entry and Replication
 
 Coronavirus virions are spherical with diameters ranging from 100 to 160 nanometers (nm).
-The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive "crown" shape for which the family of viruses was named [@isbn:0781760607; @doi:10.1128/JVI.80.3.1302–1310.2006].
+The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive "crown" shape for which the family of viruses was named [@isbn:978-0781760607; @doi:10.1128/JVI.80.3.1302–1310.2006].
```

Comment:
Not sure about this one, trying the ISBN-13

Reply:
`isbn:0781760607` and `isbn:978-0781760607` both work in my local testing